source/_components/sensor.nest.markdown

```diff
@@ -34,9 +34,12 @@ nest:
 
 By default all sensors for your available Nest devices will be monitored. Leave `monitored_conditions` blank to disable all sensors for the [Nest component](/components/nest/).
 
-Configuration variables:
-
-- **monitored_conditions** array (*Optional*): States to monitor.
+{% configuration %}
+monitored_conditions:
+  description: States to monitor.
+  required: false
+  type: list
```

Comment:
Please add the list of monitored conditions using the `keys:` option.

Reply:
The list below? How do I do that with all the sub-items that are there?